aten/src/ATen/core/union_type.cpp

```diff
@@ -227,7 +227,7 @@ UnionType::UnionType(std::vector<TypePtr> reference, TypeKind kind) : SharedType
 }
 
 UnionTypePtr UnionType::create(std::vector<TypePtr> reference) {
-  auto union_type = new UnionType(std::move(reference));
+  UnionTypePtr union_type(new UnionType(std::move(reference)));
```

Comment:
`UnionTypePtr` appears to be `std::shared_ptr<UnionType>`, so can we please use `std::make_shared`?

```
auto union_type = std::make_shared<UnionType>(std::move(reference));
```

Reply:
Of course we can